frontend/workflows/ec2/src/resize-asg.tsx

```diff
@@ -68,10 +66,12 @@ const GroupDetails: React.FC<WizardChild> = () => {
         ]}
       />
       <ButtonGroup
+        justify="flex-end"
```

Comment:
ah I just landed a PR to change the button group interface. You'll probably need to pick that up :)